collect_app/src/main/AndroidManifest.xml

```diff
@@ -124,6 +124,7 @@
         <intent-filter>
             <action android:name="android.intent.action.VIEW" />
             <action android:name="android.intent.action.EDIT" />
+            <action android:name="android.intent.action.PICK" />
```

Comment:
This is an unrelated change, please remove it from this pull request! 

You should create branches for each pull request as described in the contribution guide so that you can work on several different things at the same time without introducing this kind of change.